tests/integration/test_status.py

```diff
@@ -311,3 +311,93 @@ def test_get_builds(upstream_n_distgit, expected_results, br_list, number_of_bui
     assert table
     assert len(table.keys()) == number_of_builds
     assert table == expected_results
+
+
+@pytest.mark.parametrize(
+    "expected_releases,number_of_releases",
+    (
+        (
+            [
+                Release(
+                    "Release title1",
+                    "beer1",
+                    "0.0.1",
+                    "www.packit.dev",
+                    "23-5-2018",
+                    "www.packit.dev/tarball1",
+                ),
+                Release(
+                    "Release title2",
+                    "beer2",
+                    "0.0.2",
+                    "www.packit.dev",
+                    "23-6-2018",
+                    "www.packit.dev/tarball2",
+                ),
+                Release(
+                    "Release title3",
+                    "beer3",
+                    "0.0.3",
+                    "www.packit.dev",
+                    "23-7-2018",
+                    "www.packit.dev/tarball3",
+                ),
+            ],
+            3,
+        ),
+        (
+            [
+                Release(
+                    "Release title1",
+                    "beer1",
+                    "0.0.1",
+                    "www.packit.dev",
+                    "23-5-2018",
+                    "www.packit.dev/tarball1",
+                )
+            ],
+            1,
+        ),
+    ),
+)
+def test_get_releases(
+    upstream_instance, distgit_instance, expected_releases, number_of_releases
+):
+    u, up = upstream_instance
+    d, dg = distgit_instance
+    c = get_test_config()
+    pc = get_local_package_config(str(u))
+    up.local_project.git_project = flexmock()
+
+    flexmock(up.local_project.git_project).should_receive("get_releases").and_return(
+        expected_releases
+    )
+    status = Status(c, pc, up, dg)
+    releases = status.get_up_releases(number_of_releases)
+    assert len(releases) == number_of_releases
```

Comment:
Do we need the `number_of_releases`?

```suggestion
    assert len(releases) == len(expected_releases)
```